common.js

```diff
@@ -405,11 +405,11 @@ export function setupPath(newPathEntries) {
   return msys2Type
 }
 
-export async function setupJavaHome() {
+export async function setupJavaHome(rubyPrefix) {
   await measure("Modifying JAVA_HOME for JRuby", async () => {
     console.log("attempting to run with existing JAVA_HOME")
 
-    let ret = await exec.exec('ruby', ['--version'])
+    let ret = await exec.exec('java', ['-jar', path.join(rubyPrefix, 'lib/jruby.jar'), '--version'], {ignoreReturnCode: true})
```

Comment:
This should be `$JAVA_HOME/bin/java` (or process.env, you get the idea) as `java` in PATH might resolve to something different and in fact it does I saw `/usr/bin/java` in logs. Could you make a PR to address that?

Reply:
https://github.com/ruby/setup-ruby/pull/723